src/mods_test.go

```diff
@@ -1,37 +1,46 @@
 package main
```

Comment:
I think we can just remove this file if all the tests are commented out. Unless you want to add a few tests!